qabelbox/src/test/java/de/qabel/qabelbox/chat/view/adapters/ShareChatMessageViewHolderTest.kt

```diff
@@ -70,26 +68,38 @@ class ShareChatMessageViewHolderTest {
         val messageField: TextViewFont = mock()
         val fileField: TextViewFont = mock()
         val dateField: TextViewFont = mock()
-        val overlay : TextViewFont = mock()
-        val image : ImageView = mock()
-        val preview : ImageView = mock()
+        val overlay: TextViewFont = mock()
+        val image: ImageView = mock()
+        val preview: ImageView = mock()
+        val messageContainer: LinearLayout = mock()
+        val beginIndicator: LinearLayout = mock()
+        val beginIndicatorImage: GradientDrawable = mock()
+        stub(beginIndicator.background).toReturn(beginIndicatorImage)
+        val background = mock<Drawable>().apply {
+            stub(this.mutate()).toReturn(mock<GradientDrawable>())
+        }
+        stub(messageContainer.background).toReturn(background)
 
         stub(view.message).toReturn(messageField)
         stub(view.tvDate).toReturn(dateField)
         stub(view.file_name).toReturn(fileField)
         stub(view.msg_overlay).toReturn(overlay)
         stub(view.messageFileIcon).toReturn(image)
         stub(view.messageFilePreview).toReturn(preview)
+        stub(view.msg_container).toReturn(messageContainer)
+        stub(view.contact_avatar).toReturn(mock())
+        stub(view.chat_begin_indicator).toReturn(beginIndicator)
 
+        val fileSize = 2048L
         val holder = ShareChatMessageViewHolder(view, {})
         val msg = ChatMessage(mock<Identity>(), contact,
                 direction, Date(),
-                MessagePayloadDto.ShareMessage("text", BoxFileChatShare(status, "test.txt", 2048L,
+                MessagePayloadDto.ShareMessage("text", BoxFileChatShare(status, "test.txt", fileSize,
                         SymmetricKey(listOf()), "http://foo")))
         holder.bindTo(msg, false)
 
         verify(messageField).text = "text"
-        verify(fileField).text = "test.txt 2.0KB"
+        verify(fileField).text = "test.txt " + Formatter.formatShortFileSize(RuntimeEnvironment.application, fileSize)
```

Comment:
computation within the verification sounds like a bad Idea, how comes we no longer know the formatted filesize?